test/services/AmendReturnAnswerComparisonServiceSpec.scala

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import cacheables.ReturnDisplayApiCacheable
+import models.UserAnswers
+import models.returns.{IdDetails, ReturnDisplayApi, ReturnDisplayDetails}
+import org.scalatestplus.play.PlaySpec
+import pages.amends.{AmendDirectExportPlasticPackagingPage, AmendHumanMedicinePlasticPackagingPage, AmendImportedPlasticPackagingPage, AmendManufacturedPlasticPackagingPage, AmendRecycledPlasticPackagingPage}
+
+class AmendReturnAnswerComparisonServiceSpec extends PlaySpec {
```

Comment:
Can we add a 'complex' test where we have a actual change and a pointless change, to confirm it allows the legit

Reply:
done 👍